cpp/ql/src/experimental/Security/CWE/CWE-691/InsufficientControlFlowManagementAfterRefactoringTheCode.ql

```diff
@@ -116,4 +116,4 @@ where
   exp.(UsingArithmeticInComparison).compareWithOutZero()
   or
   exists(WhileStmt wst | wst instanceof UsingWhileAfterWhile and exp = wst.getCondition())
-select exp, "this expression needs your attention"
+select exp, "This expression needs your attention."
```

Comment:
(this message needs work, but its in experimental, and at least now the punctuation is right)

Reply:
Agreed. The message is bad, but now it follows more of the style-guide.